packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -2,9 +2,13 @@ import { expect, spy, stub } from '../../utils';
 import { SaytDriverPlugin } from '../../../src/index';
 
 describe('Sayt Driver Plugin', () => {
+  let config;
   let driver;
   let dom_events;
   let sayt;
+  let saytDataPayload;
+  let saytDataUndefinedConfig;
+  let query;
```

Comment:
The value of `query` is a primitive, so it's pointless to reassign it every time in `beforeEach` because there's no chance that it can be modified.

```suggestion
  const query = 'shirt';
```